src/index.html

```diff
@@ -1,13 +1,329 @@
 <!DOCTYPE html>
 <html lang="en">
+  <form onsubmit="this.reset(); return false;"></form>
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBiKE</title>
+    <link rel="icon" href="images/icons/favicon.svg">
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page">
+    <header class="header" id="home">
+      <div class="top-bar">
+        <a href="#home">
+          <img
+            src="images/logo-white.svg"
+            alt="MyBiKE logo"
+            class="top-bar__logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <div class="icon">
+            <a href="tel:+1 234 5555-55-55" class="icon__position icon--phone"></a>
+          </div>
+
+          <div class="icon">
+            <a href="#menu" class="icon__position icon--menu"></a>
```

Comment:
Consider using the aria-label attribute to explain the purpose of this empty link. It has a significant impact on accessibility of this page.